io.github.voxelcubes.deepqt.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.voxelcubes.deepqt
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: deepqt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-config
+  - --filesystem=xdg-cache
```

Comment:
why do you need `xdg-config` and `xdg-cache` ?

Reply:
I was under the mistaken impression that I needed those to access the xdg directories, but having removed them it still works fine, accessing the variant inside the flatpak sandbox. Fixed.